pkg/webconfig/tls_credentials.go

```diff
@@ -178,19 +178,19 @@ func (tr *tlsReferences) mountParamsForConfigmap(
 	return volumes, mounts, nil
 }
 
-// getKeyMountPath is the mount path of the private key inside a container.
-func (tr *tlsReferences) getKeyMountPath() string {
+// getKeyMountPath is the mount path of the TLS key inside a prometheus container.
+func (tr *TLSReferences) GetKeyMountPath() string {
 	secret := monitoringv1.SecretOrConfigMap{Secret: &tr.keySecret}
 	return tr.tlsPathForSelector(secret, "key")
 }
 
-// getKeyFilename returns the filename (key) of the private key.
-func (tr *tlsReferences) getKeyFilename() string {
+// getKeyFilename returns the filename (key) of the key.
```

Comment:
there's a confusion here between secret's key and private (TLS) key.
```suggestion
// getKeyFilename returns the filename of the private key.
```